app/experimenter/experiments/tests/test_generate_dashboards.py

```diff
@@ -3,7 +3,7 @@
 from django.core.management import call_command
 from django.conf import settings
 from django.test import TestCase
-from redash_client.client import RedashClient
+from stmoab.StatisticalDashboard import StatisticalDashboard
```

Comment:
It's not a good idea to have a module (file) that contains a class with the same name, becuase then you could accidentally go

from stmoab import StatisticalDashboard

dashboard = StatisticalDashboard()

and it would complain that it's not an instantiable class (it's a module) and it wouldn't be obvious to the coder at a glance why it's not working.  I usually try to stick to lower class module names, so you could call this stmoab.statistical_dashboard or something to make the distinction obvious.